python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -142,11 +195,12 @@ def load_new_trips_records(
     load_temp_for_hash_compare(db_manager, events)
     load_new_trip_data(db_manager, events)
     update_trip_stop_counts(db_manager)
+    update_static_trip_id_guess_exact(db_manager)
 
 
 # pylint: disable=R0914
 # pylint too many local variables (more than 15)
-def process_trips_table(
```

Comment:
"backup" matching is a lot more simple now. Just matching to closest static trip based on `direction_id` and `branch_route_id` or `trunk_route_id` .